gapic-generator-java/src/test/java/com/google/api/generator/gapic/composer/ComposerTest.java

```diff
@@ -154,6 +168,22 @@ public void composeSamples_parseProtoPackage() {
     }
   }
 
+  @Test
+  public void testEmptyGapicContext_succeeds() {
```

Comment:
From what I read from the test, you want to verify no exception is thrown in `Composer.composeServiceClasses(GapicContext.EMPTY)`

How about write tests verifying the result, rather assert the exception is null?

Reply:
Thanks, this simplifies the test. Done.